spec/models/item_spec.rb

```diff
@@ -5,8 +5,19 @@
                 item.categories << category
                 item.save
                 item}
+  let(:item2) { item = Fabricate.build(:item)
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.